pyoozie/tags.py

```diff
@@ -619,6 +619,73 @@ def _xml(self, doc, tag, text, on_next, on_error):
         return doc
 
 
+class Serial(_AbstractWorkflowEntity):
+    """Sequence of entities to execute (implemented by chaining entities and 'OK' transitions)"""
+
+    def __init__(self, *entities, **kwargs):
+        # type: (*_AbstractWorkflowEntity, **_AbstractWorkflowEntity) -> None
+        super(Serial, self).__init__(on_error=kwargs.get(str('on_error')))
+        self.__entities = tuple(copy.deepcopy(entities))  # type: typing.Tuple[_AbstractWorkflowEntity, ...]
+
+    def identifier(self, xml_tag=None):  # type: (typing.Optional[typing.Text]) -> typing.Text
+        return self.__entities[0].identifier(xml_tag) if self.__entities else None
```

Comment:
This is just a sequence, so let's address it by the entry point for this sequence, the first real tag.